src/test/php/PDepend/Bugs/ParserBug713Test.php

```diff
@@ -11,7 +11,7 @@
  * Test case for bug #713.
  *
  * @ticket 713
- * @covers \PDepend\Source\Language\PHP\AbstractPHPParser::parseIssetExpression
+ * @covers \PDepend\Source\Language\PHP\AbstractPHPParser::parseVariableList
```

Comment:
I think the test wants to and does cover both methods.

```suggestion
 * @covers \PDepend\Source\Language\PHP\AbstractPHPParser::parseIssetExpression
 * @covers \PDepend\Source\Language\PHP\AbstractPHPParser::parseVariableList
```

Reply:
That works too